runtime/compiler/codegen/J9CodeGenerator.hpp

```diff
@@ -158,23 +158,23 @@ class OMR_EXTENSIBLE CodeGenerator : public OMR::CodeGeneratorConnector
                                           uint8_t *target,
                                           uint8_t *target2,  //pass in NULL when no target2
                                           TR_ExternalRelocationTargetKind kind,
-                                          char *generatingFileName,
+                                          const char *generatingFileName,
                                           uintptr_t generatingLineNumber,
                                           TR::Node *node);
    //TR::ExternalOrderedPair32BitRelocation
    void addProjectSpecializedPairRelocation(uint8_t *location1,
                                           uint8_t *location2,
                                           uint8_t *target,
                                           TR_ExternalRelocationTargetKind kind,
-                                          char *generatingFileName,
+                                          const char *generatingFileName,
                                           uintptr_t generatingLineNumber,
                                           TR::Node *node);
    //TR::BeforeBinaryEncodingExternalRelocation
    void addProjectSpecializedRelocation(TR::Instruction *instr,
                                           uint8_t *target,
                                           uint8_t *target2,   //pass in NULL when no target2
                                           TR_ExternalRelocationTargetKind kind,
-                                          char *generatingFileName,
+                                          const char *generatingFileName,
```

Comment:
There are declarations of `OMR::CodeGenerator::addProjectSpecializedRelocation` and `OMR::CodeGenerator::addProjectSpecializedPairRelocation` upstream in [`OMRCodeGenerator.hpp](https://github.com/eclipse/omr/blob/149b5245ac056f5025d141760ea995822fc30c74/compiler/codegen/OMRCodeGenerator.hpp)`.  Will any compilers complain about the fact that the argument is `const char *` in one place but not the other?  Regardless, it might be more clear if both are consistent.

Reply:
Thanks for the catch! I made a corresponding update to https://github.com/eclipse/omr/pull/7185.

Reply:
I think that with this and the corresponding change to eclipse/omr#7185, the two pull requests will need a co-ordinated merge - or at least, it would be safer that way.  May I ask you to update the description of this pull request to note the dependency?